### PR TITLE
add / remove users to a room

### PR DESCRIPTION
**Pull request by mludi**
### What?

You can add and remove users by their id to a room

### Why?

I'll cover all functions from the official docs in this sdk.

### CODE REVIEW

File: src/Chatkit.php [hunk 1]
```diff
@@ -347,6 +347,91 @@ public function joinRoom($options)
...
+        if (!isset($options['user_ids'])) {
+            throw new MissingArgumentException('You must provide a User ID(s) to add them to a room');
+        }
+
+        $roomId = $options['room_id'];
+
+        $body = [];
+        $body['user_ids'] = $options['user_ids'];
+
+        $token = $this->generateToken([
```
Comment (hamchapman) [inline src/Chatkit.php]: When you want a token with the `'su' => true` claim you can instead just call `$this->getServerToken()`

File: src/Chatkit.php [hunk 2]
```diff
@@ -347,6 +347,91 @@ public function joinRoom($options)
...
+        if (!isset($options['user_ids'])) {
+            throw new MissingArgumentException('You must provide a User ID(s) to remove them from a room');
+        }
+
+        $roomId = $options['room_id'];
+
+        $body = [];
+        $body['user_ids'] = $options['user_ids'];
+
+        $token = $this->generateToken([
```
Comment (hamchapman) [inline src/Chatkit.php]: Again here you can instead use `$this->getServerToken()`

**Review by hamchapman — APPROVED** [review]
Awesome - thanks again for this!

I left a comment about being able to use `getServerToken()` as a "shortcut" in a couple of places, but otherwise this looks great.

**Comment by mludi** [comment]
HI @hamchapman I made another PR which will use the getServerToken() :)

**Pull request closed without merging**